.github/workflows/main.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - '2.5'
+          - '2.7'
```

Comment:
Since you have a PR to add this support, seems reasonable to drop it here and then add it in #113